pkg/otlp/metrics/metrics_translator.go

```diff
@@ -54,6 +55,12 @@ type Translator struct {
 	cfg     translatorConfig
 }
 
+// RuntimeMetricsTelemetry provides runtime metrics information on a MapMetrics run
+type RuntimeMetricsTelemetry struct {
```

Comment:
I would rename this to `Metadata` and document it as such:
```go
// Metadata specifies information about the outcome of the MapMetrics call.
type Metadata struct {
    // RuntimeMetrics reports whether runtime metrics where found during metrics mapping.
    RuntimeMetrics bool
    // Languages specifies a list of languages for which runtime metrics where found.
    Languages []string
}
```
Lastly -- why do you need `HasRuntimeMetrics`. Isn't doing `len(Languages) > 0` the same thing? If yes, please remove the boolean. And if yes, you may as well just return a set of tags. But keeping the struct is also a good option because I suspect we may want to return more metadata eventually.